suite-common/message-system/src/messageSystemThunks.ts

```diff
@@ -22,7 +22,18 @@ import {
 } from './messageSystemSelectors';
 import { jws as configJwsLocal } from '../files/config.v1';
 
+// this is for local development purpose
+// set to true to always fetch local jws
```

Comment:
Nitpick, feel free to ignore
```suggestion
// Enable this for local development purposes:
// set to true to always fetch local JWS
```

Reply:
Great idea: 88529b1dbd